### PR TITLE
[Applab Dataset] Enable importing static datasets

### PR DESCRIPTION
**Pull request by ajpal**
When you click the import button for a static dataset, we copy the data from the shared firebase channel into the user's project channel. After that, it is the same as any other table the user created- they can modify it however they want and it is not linked to the original data.

### CODE REVIEW

File: apps/src/storage/dataBrowser/LibraryTable.jsx [hunk 1]
```diff
@@ -25,6 +27,22 @@ class LibraryTable extends React.Component {
       collapsed: !this.state.collapsed
     });
 
+  importTable = () => {
+    if (this.props.current) {
+      // TODO
```
Comment (jmkulwik) [inline apps/src/storage/dataBrowser/LibraryTable.jsx]: Can you link a task or add a bit more to the description?

File: apps/src/storage/firebaseStorage.js [hunk 2]
```diff
@@ -455,6 +456,33 @@ FirebaseStorage.resetForTesting = function() {
...
+    })
+    .then(() => {
+      return getSharedDatabase()
+        .child(`storage/tables/${tableName}/records`)
+        .once('value');
+    })
+    .then(snapshot => {
+      getRecordsRef(tableName).set(snapshot.val());
+    })
+    .then(onSuccess, onError);
```
Comment (jmkulwik) [inline apps/src/storage/firebaseStorage.js:483]: Should errors be handled per operation?

Comment (ajpal) [reply apps/src/storage/firebaseStorage.js:483]: I think just checking for error at the end is fine. At most we might want to show a "something went wrong" dialog to the user if the import fails for whatever reason, but I don't think the granularity of knowing which step failed would necessarily be useful.

We can always add more error cases later, but trying to keep things simple/mvp for now

File: apps/src/storage/dataBrowser/LibraryTable.jsx [hunk 3]
```diff
@@ -25,6 +27,22 @@ class LibraryTable extends React.Component {
...
     });
 
+  importTable = () => {
+    if (this.props.current) {
+      // TODO: Implement current tables (see STAR-615)
+    } else {
+      FirebaseStorage.copyStaticTable(
+        this.props.name,
+        () => {
+          console.log('success');
```
Comment (epeach) [inline apps/src/storage/dataBrowser/LibraryTable.jsx:42]: Is there a plan to update these success/error responses?

Comment (ajpal) [reply apps/src/storage/dataBrowser/LibraryTable.jsx:42]: Yes, we will probably want to make some UI corresponding to success/error. I don't think that flow is finalized yet, though.

Comment (epeach) [reply apps/src/storage/dataBrowser/LibraryTable.jsx:42]: Between this and refactoring the css, there seem to be a few tasks to loop back to when the UI is finalized. Is there a place we are tracking those items to make sure they don't get lost?

Comment (ajpal) [reply apps/src/storage/dataBrowser/LibraryTable.jsx:42]: Filed https://codedotorg.atlassian.net/browse/STAR-671 and https://codedotorg.atlassian.net/browse/STAR-672 to track

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**